### DISCUSSION TITLE
williams system 7?

### DISCUSSION THREAD

**Discussion by gr3yh47** [missionpinball/missionpinball#37] [Q&A]
is there a ready-to-go hardware interface for MPF to control williams system 7?

I want to reprogram black knight (1980)

**Comment by kylenahas** [comment]
I'm currently working on a board to do just this, but it's still just in the prototype phase, sorry. It'll be called the Legacy Pinball Classics Controller, and the goal is to basically just replace the MC6808 on the any Williams MPU Board.
